spec/status.xml

```diff
@@ -5,62 +5,47 @@
 time of its publication. Other documents may supersede this document.
 A list of current W3C publications and the latest revision of this
 technical report can be found in the <loc
-href="http://www.w3.org/TR/">W3C technical reports index</loc> at
-http://www.w3.org/TR/.</emph></p>
+href="https://www.w3.org/TR/">W3C technical reports index</loc> at
+https://www.w3.org/TR/.</emph></p>
 
-<p>This document has been reviewed by W3C Members, by software developers,
-and by other W3C groups and interested parties, and is endorsed by the Director
-as a W3C Recommendation. It is a stable document and may be used as reference
-material or cited from another document. W3C's role in making the Recommendation
-is to draw attention to the specification and to promote its widespread
-deployment. This enhances the functionality and interoperability of the Web.</p>
+<p>This document was published by the <loc href="http://www.w3.org/AudioVideo/TT/">Timed Text Working Group</loc> as a Candidate Recommendation. 
+This document is intended to become a W3C Recommendation. Comments regarding this document are welcome. 
+Please send them to <loc href="mailto:public-tt@w3.org?subject=%5Bttml1%5D">public-tt@w3.org</loc> 
+(<loc href="mailto:public-tt-request@w3.org?subject=subscribe">subscribe</loc>,
+<loc href="https://lists.w3.org/Archives/Public/public-tt/">archives</loc>) with <code>[ttml1]</code> at the start of your email's subject.
+W3C publishes a Candidate Recommendation to indicate that the document is believed to be stable 
+and to encourage implementation by the developer community. This Candidate Recommendation is expected to 
+advance to Proposed Recommendation no earlier than &crEnd;.</p>
 
-<p>This version reflects changes made as a result of applying errata and addressing
-comments received since the Recommendation was first published. With one minor
-exception noted below, these changes are editorial.</p>
+<p>This edition of the Timed Text Markup Language (TTML) 1.0
+<bibref ref="ttml10"/> clarifies ambiguities and corrects errors identified in the 
+<loc href="&prevrec;">previous
+edition</loc>, which this edition supersedes. These clarifications and corrections are intended to 
+reflect current implementations, and, as such, no implementation report is provided. 
+In addition, no features were added or removed.</p>
```

Comment:
For consistency I'd word the last sentence here as "No feature is introduced or removed in this edition."

The rest of the paragraph is in present tense, and the "In addition" is not needed in my view.